common/src/state/mod.rs

```diff
@@ -323,7 +323,14 @@ impl State {
     }
 
     pub fn process_warp_event(&mut self, event: WarpEvent) {
-        log::debug!("process_warp_event: {event}");
+        // Blink events are very frequent
+        if !matches!(
+            event,
+            WarpEvent::Blink(BlinkEventKind::ParticipantSpeaking { peer_id: _ })
+                | WarpEvent::Blink(BlinkEventKind::SelfSpeaking)
+        ) {
+            log::debug!("process_warp_event: {event}");
```

Comment:
i'm not sure what the question is but the answer is probably no :grin:

Reply:
I believe he referring to replacing `process_warp_event` with `process_blink_event`, however I think `process_warp_event` is appropriate since blink is still apart of warp.